ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/engine/EngineNewPayloadV3.java

```diff
@@ -52,20 +51,12 @@ public String getName() {
 
   @Override
   protected ValidationResult<RpcErrorType> validateParameters(
-      final EnginePayloadParameter payloadParameter,
-      final Optional<List<String>> maybeVersionedHashParam,
-      final Optional<String> maybeBeaconBlockRootParam) {
-    if (payloadParameter.getBlobGasUsed() == null || payloadParameter.getExcessBlobGas() == null) {
+      final EngineNewPayloadRequestParameter params) {
+    if (params.getExecutionPayload().getBlobGasUsed() == null
+        || params.getExecutionPayload().getExcessBlobGas() == null) {
```

Comment:
`maybeVersionedHashParam` == null would never happen to be true in the previous code as maybeVersionedHashParam was being extracted using `getOptionalList` which returns Optional.Empty in case of `null`.

maybeVersionedHashParam is now treated as a required parameter for V3 and will throw if is missing.  `BeaconBlockRoot` will also throw if `null`. 

There are two tests in this PR that test this validation.